ext/src/ruby_api/instance.rs

```diff
@@ -89,15 +93,15 @@ impl Instance {
     /// @def exports
     /// @return [Hash{String => Func, Memory}]
```

Comment:
We're gonna have to update that signature, along with `Instance#export` and `Linker#get`.

I can take care of it once it's merged though.